tests/components/wled/test_config_flow.py

```diff
@@ -131,10 +132,8 @@
     flow = config_flow.WLEDFlowHandler()
     flow.hass = hass
     flow.context = {"source": SOURCE_USER}
-    result = await flow.async_step_user({CONF_HOST: "example.local"})
-
-    assert result["reason"] == "already_configured"
-    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
+    with pytest.raises(data_entry_flow.AbortFlow):
+        await flow.async_step_user({CONF_HOST: "example.local"})
```

Comment:
If we change to use the core interface `hass.config_entries.flow.async_init` this test will be more robust and will still return the same result as before.

Reply:
Done. (while out of scope, changed some other tests as well).